src/types.json

```diff
@@ -27,6 +27,20 @@
       "object": "products and services usage"
     }
   },
+  "Optional Service Terms": {
```

Comment:
I don't understand the meaning of this document type. Can you explain please? :)

Reply:
This document stands for optional services/subservices potentially useful when using the main service, such as those quoted in AdSense TOS : 
```
16. Service-Specific Terms

If you choose to implement any of the following Services on a Property, you also agree to the additional terms identified below:

AdMob: the AdMob Publisher Guidelines and Policies.

Custom Search Engine: the Custom Search Engine Terms of Service.
```
I wasn't sure of the name though

Reply:
I think this document type is too non-specific and I fear that it becomes a catch-all type. So I'm in favor of removing this it.

Reply:
Do you want me to find another document type or do we put aside those documents for the time being ?

Reply:
Put aside those documents for now.

Reply:
Done, I rebased onto master :)